tests/appsec/test_telemety.py

```diff
@@ -23,23 +23,23 @@
 
 @pytest.fixture
 def mock_telemetry_metrics_writer():
+    telemetry_metrics_writer.disable()
```

Comment:
Why do we need to make this change?

Reply:
Because Telemetry metrics is working in all test scenarios in background, `telemetry metrics writer.disable()` flushes the queue before the telemetry tests start